src/ZEO/ClientStorage.py

```diff
@@ -53,6 +53,9 @@
 
 logger = logging.getLogger(__name__)
 
+# max signed 64-bit value ~ infinity :) Signed cuz LBTree and TimeStamp
+m64 = b'\x7f\xff\xff\xff\xff\xff\xff\xff'
```

Comment:
No.  All I did there, for now, was to fix loadBefore, not implement load in terms of it.